src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -398,6 +399,7 @@ struct blk_checkpoint_header
   uint64_t     num_signatures;
 };
 static_assert(sizeof(blk_checkpoint_header) == 2*sizeof(uint64_t) + sizeof(crypto::hash), "blk_checkpoint_header has unexpected padding");
+static_assert(sizeof(service_nodes::voter_to_signature) == sizeof(uint16_t) + 6 /*padding*/ + sizeof(crypto::signature), "Unexpected padding/struct size change. DB checkpoint signature entries need to be re-migrated to the new size");
```

Comment:
I don't think this will pass on 32-bit arches, which will only have 2 bytes of padding (because the crypto primitives alignment is the same as size_t, which is 4-bytes on 32-bit arches).

Reply:
One way to fix it is to keep this static assert and explicitly add padding by adding a member:
```C++
char /*padding*/ : 48;
```
 into `voter_to_signature` between `voter_index` and `signature`.

Reply:
Added